bekeplar/EPIC-Mail
# Challenge one feedback

## What this PR does?
- Refactoring some code snippets
- Adding some Javascript functions to the application
- Minimizing colors
- Add logo to every page of the app
- Removes redundancy from login page 
- Adds navigation bar/ link to login on signup page
- Improving the user interface
- Removes unwanted buttons and headers
- Remove commented code

## Tasks completed
- Writing some new HTML for the pages
- Added javascript functions to make more user-friendly UI
- Refactoring most of the CSS in the project
- Adds a navigation bar to separate inbox, sent and draft messages

## How should this be manually tested?
On this Pull- request navigate to the files changed, tab to see the various files changed 

## Pivotal stories attached to the PR
[https://www.pivotaltracker.com/n/projects/2318783]

kindly review and give me feedback on necessary improvements
@Emmanuelaaron, @alexxsanya, @nakimera, @kyakusahmed, @neelxie
